text/0000-bionic-mixins.md

```diff
@@ -0,0 +1,85 @@
+# Meta
+[meta]: #meta
+- Name: Defining Mixins for io.buildpacks.stacks.bionic
+- Start Date: 2019-12-05
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: (put "N/A" unless this replaces an existing RFC, then link to that RFC)
+
+# Summary
+[summary]: #summary
+
+The Platform API specification allows stack maintainers to define stack mixins for a given stack.
+The Cloud Native Buildpacks project informally defines stack build and run images with ID `io.buildpacks.stacks.bionic` as containing exactly the set of Ubuntu packages in the `ubuntu:bionic` image on Docker Hub.
+This RFC proposes that we formalize the existing definition of `io.buildpacks.stacks.bionic` and define mixins on that stack to be Ubuntu 18.04 LTS packages. 
+
+In general, a stack is a contract that should be more like "Ubuntu Bionic" and less like "Platform XYZ's Re-distribution of Ubuntu Binoic."
+Stack images can still be provided by platforms/vendors, but they should use the standard CNB stack contracts where applicable.
+
+# Proposal
+[proposal]: #proposal
+
+This RFC proposes that we define `io.buildpacks.stacks.bionic` such that:
+1. The base set of packages is `ubuntu:bionic` with `ca-certificates`, `libssl1.1`, and `openssl` added.
+2. Mixins without an `=` are Ubuntu 18.04 LTS package names without version or architecture info from official package mirrors.
+3. Mixins that begin with `set=` are defined by the project as such:
+  a. `build:set=shell-utils` represents the change set: `build:curl`, `build:jq`, and yj CLI at build-time.
```

Comment:
```suggestion
  a. `build:set=shell-utils` represents the change set: `build:curl`, `build:jq`, and `build:yj` at build-time.
```

Reply:
yj is not available as an Ubuntu package, so I went with "yj CLI at build-time" instead of "`build:yj`."

Maybe it's worth defining `build:set=yj` as well?